core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -1525,9 +1525,7 @@
    */
   public static final long HANDOFF_SIZE_DEFAULT = 1024L * 1024 * 1024;
 
-  public static final String TIMESERIES_EVENTTIME = "timeseries.eventtime";
-
-  public static final String TIMESERIES_HIERARCHY = "timeseries.hierarchy";
+  public static final String TIMESERIES_EVENTTIME = "event_time";
```

Comment:
Declare the constant in datamap class, not in CarbonCommonConstaints

Reply:
ok, move it to TimeSeriesUtil